build/v1/types.go

```diff
@@ -938,6 +938,11 @@ type BuildConfigSpec struct {
 	// When a BuildConfig is created, the 5 most recent failed builds are retained unless this value is set.
 	// If removed after the BuildConfig has been created, all failed builds are retained.
 	FailedBuildsHistoryLimit *int32 `json:"failedBuildsHistoryLimit,omitempty" protobuf:"varint,5,opt,name=failedBuildsHistoryLimit"`
+
+	// UseClusterCA denotes whether the builder container will use CAs from the cluster, or the CAs included in the builder image's own /etc/pki/ca-trust directory.
+	// Setting this to true will overwrite the contents of /etc/pki/ca-trust w/ the CA trust values from the cluster. Default: false
+	// +optional
+	UseClusterCA bool `json:"useClusterCA,omitempty" protobuf:"varint,6,opt,name=useClusterCA"`
```

Comment:
Do we _ever_ want to support default and override values for this field?

If yes, then this needs to be a pointer so we can determine user intent.

Reply:
I checked with @bparees about this before doing it and he said it doesn't need to be a pointer.